Assets/Scripts/Ricochet.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[CreateAssetMenu(fileName = "RicochetWeapon", menuName = "RicochetWeapon")]
```

Comment:
Please change this to 
`[CreateAssetMenu(fileName = "Ricochet", menuName = "LTG5/Weapons/Ricochet")]`